src/VisualStudio/IntegrationTest/New.IntegrationTests/VisualBasic/BasicIntelliSense.cs

```diff
@@ -31,6 +31,9 @@ public override async Task InitializeAsync()
         {
             await base.InitializeAsync();
 
+            // Try disable the responsive completion option again: https://github.com/dotnet/roslyn/issues/70787
+            await TestServices.StateReset.DisableResponsiveCompletion(HangMitigatingCancellationToken);
```

Comment:
😕 I don't understand how this would make a difference. `InitializeAsync` _is_ the call which disables responsive non-completion.

Reply:
Yea, I don't know. That's tracked by the bug I filed, just want to get this in while we are trying to figure it out